docs/transports.md

```diff
@@ -240,6 +241,19 @@ how to setup [Kibana][kibana].
 For Docker users, see
 https://github.com/deviantony/docker-elk to setup an ELK stack.
 
+<a id="pino-stackdriver"></a>
+### pino-stackdriver
+The [pino-stackdriver](https://www.npmjs.com/package/pino-stackdriver) module is a transport that will forward logs to the [Google Stackdriver](https://cloud.google.com/logging/) log service through it's API.
+
+Given an application `foo` that logs via pino, a stackdriver log project `bar` and credentials in the file `/credentials.json`, you would use `pino-stackdriver`
+like so:
+
+``` sh
+$ node foo | pino-stackdriver --project bar --credentials /credentials.json
+```
+
+for full documentation of command line switches read [readme](https://github.com/ovhemert/pino-stackdriver#readme)
```

Comment:
```suggestion
For full documentation of command line switches read [readme](https://github.com/ovhemert/pino-stackdriver#readme)
```

Reply:
Oh no... let's hope Travis will not fail on that change ;)

Thanks 👍